frontend/src/global_styles/content/work_packages/tabs/_activities.sass

```diff
@@ -68,6 +68,9 @@
 .comment ul
   list-style-type: disc !important
 
+  &.todo-list
+    list-style: none !important
+
```

Comment:
Could you please check whether we can remove these lines all together? I see no reason to always enforce a `disc` style for bullet points in the comments, especially since this is different in other fields. But maybe I am missing something...